packages/augur-test/tests/state/api/Router.test.ts

```diff
@@ -0,0 +1,49 @@
+import { API } from "@augurproject/state/src/api/API";
+import { Augur } from "@augurproject/api";
+import { DB } from "@augurproject/state/src/db/DB";
+import { Router } from "@augurproject/state/src/api/Router";
+import { ethers } from "ethers";
+import { makeTestAugur, ACCOUNTS, makeDbMock } from "../../../libs";
+
+const mock = makeDbMock();
+
+beforeEach(async () => {
+  mock.cancelFail();
```

Comment:
I've been putting this in every test but it's not actually needed if you aren't telling the mockdb to fail.

Reply:
Removed. Thanks.